content/en/docs/concepts/architecture/nodes.md

```diff
@@ -518,9 +518,10 @@ During a non-graceful shutdown, Pods are terminated in the two phases:
 {{< feature-state state="beta" for_k8s_version="v1.28" >}}
 
 To enable swap on a node, the `NodeSwap` feature gate must be enabled on
-the kubelet, and the `--fail-swap-on` command line flag or `failSwapOn`
+the kubelet (default is true), and the `--fail-swap-on` command line flag or `failSwapOn`
 [configuration setting](/docs/reference/config-api/kubelet-config.v1beta1/)
-must be set to false.
+must be set to false. 
+To allow Pods to utilize swap, `swapBehavior` should be set to `LimitedSwap` in the kubelet config.
```

Comment:
Currently `LimitedSwap` is the only swap behavior that allows pods to utilize swap. However, in the future we may want to add more behaviors.

Do you think we can instead write something like:
```suggestion
To allow Pods to utilize swap, `swapBehavior` should not be set to `NoSwap` (which is the default behavior) in the kubelet config.
```